src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1202,19 +1202,72 @@ private async Task<Size> CalculateStorageSize(string databaseName)
         public async Task SetUnusedDatabaseIds()
         {
             var database = GetStringQueryString("name");
+            var validate = GetBoolValueQueryString("validate", required: false);
```

Comment:
I think the default here should be `true`

Reply:
After thinking about this, ignore this comment :)
We will maybe change the default as part of the studio work on this

Reply:
Done